hkl/tests/test_tardis.py

```diff
@@ -51,10 +24,12 @@ def kcf_sample(tardis):
 
     # lengths must have same units as wavelength
     # angles are in degrees
-    lattice = Lattice(a=5.857, b=5.857, c=7.849, alpha=90.0, beta=90.0, gamma=90.0)
+    a0 = 5.857
+    c0 = 7.849
+    tetragonal = (a0, a0, c0, 90, 90, 90)
```

Comment:
```suggestion
    tetragonal = (a0, a0, c0, 90., 90., 90.)
```

Probably useful down the line to already make the angles `float` values here.